lighter/modules/blockchain.py

```
@@ -1318,6 +1323,18 @@ def _send_eth_transaction(
                         error = inner_error
                     else:
                         break
+                elif "intrinsic gas too low" in str(error):
+                    try:
+                        retry_count += 1
+                        options["gas"] += options["gas"]
+                        signed = self._sign_tx(method, options)
+                        tx_hash = self.web3.eth.send_raw_transaction(
+                            signed.rawTransaction,
+                        )
+                    except ValueError as inner_error:
+                        error = inner_error
+                    else:
+                        break
```

I think we should throw an error in this case, if they entered a specific gas limit client side should decide if they want to double the gas limit or not. Since they are actually spending money